csrc/ir/utils.cpp

```diff
@@ -858,6 +858,7 @@ class ValidateDomainEquivalence : private IterVisitor {
               derived_domain.end(),
               [&](auto id) {
                 return id->getIterType() == IterType::Symbolic ||
+                    id->getIterType() == IterType::Broadcast ||
```

Comment:
@jacobhinkle I don't think I'm doing the right thing here. But it asserts here with the changes... I think I missed something else.